client/src/pages/admin/PatientProfile/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { get } from 'axios';
+import './style.css';
+import { useParams } from 'react-router-dom';
+
+import Title from '../../../components/common/Title';
+import PatientDetailsForm from '../../../components/PatientDetailsForm';
+import PatientTreatmentForm from '../../../components/PatientTreatmentForm';
+import PatientHistory from '../../../components/PatientHistory';
+import Loading from '../../../components/common/Loading';
+import AlertMessage from '../../../components/common/AlertMessage';
+
+function PatientProfile() {
+  const [historyData, setHistoryData] = useState([]);
+  const [profileData, setProfileData] = useState([]);
+  const [updateDate, setUpdateDate] = useState(0);
+  const [balanceValue, setBalanceValue] = useState(0);
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const { patientId } = useParams();
+  useEffect(() => {
+    const getPatientHistory = async () => {
+      try {
+        const {
+          data: {
+            data: { profile, history, balance },
+          },
+        } = await get(`/api/v1/patients/${patientId}`);
+        setLoading(false);
+        setHistoryData(history);
+        setProfileData(profile);
+        return setBalanceValue(balance);
+      } catch (err) {
+        setLoading(false);
+        if (err.response) {
+          const {
+            response: { data },
+          } = err;
+          return setErrorMessage(data.message ? data.message : data);
+        }
+        return setErrorMessage(err);
+      }
+    };
```

Comment:
Move this function to outside the useEffect

Reply:
Done